src/eth/homestead_transaction.rs

```diff
@@ -29,6 +33,92 @@ pub struct EthLegacyHomesteadTxArgs {
 }
 
 impl EthLegacyHomesteadTxArgs {
+    /// Returns legacy homestead transaction signature
+    pub fn signature(&self) -> anyhow::Result<Signature> {
+        // Check if v is either 27 or 28
+        ensure!(
+            self.v == BigInt::from(27) || self.v == BigInt::from(28),
```

Comment:
Those numbers occur extensively across the codebase - they should be put as constants.

Reply:
fixed